Makefile

```diff
@@ -145,6 +145,24 @@ release-install:
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lncli
 
+#? cross-release-install: Build lnd and lncli release binaries for single/all
+# supported platforms to /tmp (useful for checking cross compilation or priming
```

Comment:
I think this help text only works if everything is on one line. See `make help`.
So this file is exempt from the 80 char limit.

Reply:
Fixed also for other commands in a second commit.